### PR TITLE
frontend: better moonpay fees

### PR DESCRIPTION
**Pull request by thisconnect**
Updated the Fees for credit & debit cards and Apple/Google/Samsung.

### CODE REVIEW

File: frontends/web/src/routes/buy/info.tsx [hunk 1]
```diff
@@ -151,12 +151,12 @@ class BuyInfo extends Component<Props, State> {
                                                 </tr>
                                                 <tr>
                                                     <td>{t('buy.info.disclaimer.payment.table.2_method')}</td>
```
Comment (x1ddos) [inline frontends/web/src/routes/buy/info.tsx:153]: I have to say it's not immediately clear which payment method this is by looking at the code... A better name would've been `cc_method`, `mobile_method`, `bank_method`, etc.

File: frontends/web/src/routes/buy/info.tsx [hunk 2]
```diff
@@ -151,12 +151,12 @@ class BuyInfo extends Component<Props, State> {
...
-                                                    <td class={style.nowrap}>5.9 %</td>
+                                                    <td class={style.nowrap}>4.9 %</td>
                                                     <td>{t('buy.info.disclaimer.payment.table.2_description')}</td>
                                                 </tr>
                                                 <tr>
                                                     <td>{t('buy.info.disclaimer.payment.table.3_method')}</td>
```
Comment (x1ddos) [inline frontends/web/src/routes/buy/info.tsx:158]: This is apple/google/samsung pay, right. Did we try and know it is actually working?

**Review by x1ddos — APPROVED** [review]
LGTM

File: frontends/web/src/routes/buy/info.tsx (hunk 1 above)
Comment (thisconnect) [reply frontends/web/src/routes/buy/info.tsx:153]: Sorry I only saw LGTM and merged...

File: frontends/web/src/routes/buy/info.tsx (hunk 2 above)
Comment (x1ddos) [reply frontends/web/src/routes/buy/info.tsx:158]: No worries, it was for the future. Remember next time :)

File: frontends/web/src/routes/buy/info.tsx (hunk 1 above)
Comment (x1ddos) [reply frontends/web/src/routes/buy/info.tsx:153]: No worries, it was for the future. Remember next time :)

**Pull request merged**